tempodb/encoding/vparquet3/schema.go

```diff
@@ -150,16 +150,27 @@ type DedicatedAttributes struct {
 	String10 *string `parquet:",snappy,optional,dict"`
 }
 
-type EventAttribute struct {
-	Key   string `parquet:",snappy,dict"`
-	Value []byte `parquet:",snappy"` // Was json-encoded data, is now proto encoded data
+type Event struct {
+	TimeSinceStartUnixNano uint64      `parquet:",delta"`
+	Name                   string      `parquet:",snappy"`
+	Attrs                  []Attribute `parquet:",list"`
+	DroppedAttributesCount int32       `parquet:",snappy"`
```

Comment:
`DroppedAttributesCount` is zero for most spans, links, resources, and events. It would be interesting to run benchmarks with `parquet:",snappy,delta"` for all dropped attribute count columns.